src/Makefile

```diff
@@ -61,7 +61,7 @@ mlnx/swss_1.0.0_amd64.deb mlnx/intfsyncd mlnx/neighsyncd mlnx/orchagent mlnx/por
 	cp sonic-swss/debian/swss/usr/bin/fpmsyncd .
 	
 $(addprefix sonic-linux-kernel/,linux-headers-3.16.0-4-amd64_3.16.7-ckt11-2+acs8u2_amd64.deb linux-headers-3.16.0-4-common_3.16.7-ckt11-2+acs8u2_amd64.deb linux-image-3.16.0-4-amd64-dbg_3.16.7-ckt11-2+acs8u2_amd64.deb linux-image-3.16.0-4-amd64_3.16.7-ckt11-2+acs8u2_amd64.deb xen-linux-system-3.16.0-4-amd64_3.16.7-ckt11-2+acs8u2_amd64.deb):
-	pushd sonic-linux-kernel; sudo ./build.sh; popd
+	pushd sonic-linux-kernel; sudo ./build.sh $(LINUX_PATCH); popd
```

Comment:
suggest drop this for now. Let's first agree on how the sonic kernel is going to be build in the other repo.